### DISCUSSION TITLE
test dataset

### DISCUSSION THREAD

**Discussion by zhjw0927** [STVIR/pysot#409] [General]
I'm new to tracking, and I have a question about the test data.
In test, OTB2015 VOT16/17/18 are supported, but in train, COCO DET VID are supported.
What's the connection between these?
If I want to run train, what data should I use?
Test?
Thank you!

**Comment by ZhiyuanChen** [comment]
COCO DET VID are training dataset
OTB2015 VOT16/17/18 are evaluation

**Comment by zhjw0927** [comment]
Thank you! I got it.
